programs/lib/linker/flexpret.ld

```diff
@@ -24,8 +24,8 @@ SECTIONS
     .data :
     {
         __data_start__ = .;
+        *(.rodata .rodata*)
```

Comment:
@erlingrj  did we ever resolve why this works? Would it be possible to make a test that fails with the previous configuration?